microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -70,164 +71,274 @@ After doing the above steps, the extension is ready to be submitted via Partner
+| **Support contact details** | - | The URL to your support webpage, or the email address to contact your support team. |
+| **Mature content** | - | Checkbox to specify if your extension includes mature content.  Extension rating helps determine the appropriate age group of the target audience of your extension.  To help determine if your extension has mature content, see [Developer policies for the Microsoft Edge Add-ons store](/legal/microsoft-edge/extensions/developer-policies). |
 
-Select **Save & Continue** to continue to the **Store listings** section.
+Properties that are not indicated as Required are Optional.
 
-> [!IMPORTANT]
-> Your developer/organization name, website URL, and support contact details that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+Your developer/organization name, **Website URL**, and **Support contact details** that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+
+After filling in the **Properties** page for the extension, click the **Save & Continue** button.
+
+The **Store listings** page opens.
 
 
 <!-- ========================================================================== -->
 ## Step 5: Add Store listing details for your extension
 
-The information provided in the following section is displayed to users who review your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.  To list your extension in the store, the following details are required.
+1. In the **Store Listings** page, click the **Edit details** button:
+
+![Store listing section for an extension](./publish-extension-images/store-listing-section-for-extension.png)
+
+
+The information provided in the **Store listings** page is displayed to users who view your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.
+
+In the **Language details** section, fill in the following properties.
+
+| Property name | Required? | Description |
+|---|---|---|
+| **Display name** | Required for at least one language | The `name` of your extension specified in the manifest file of your extension.  To change the store display name after submission, you can update the name in the manifest file, create a new extension package, and then re-upload it. |
+| **Description** | Required for each language | The `description` field explains what your extension does, why users should install it, and any other relevant information users need to know.  <br/>* **Length:** Minimum 250 characters, maximum 5000 characters.  <br/>* **Clarity:** Must clearly describe the full functionality of your extension.  <br/>* **Completeness:** Should be thorough and should leave no important details out.  <br/>* **Appropriateness:** Must not contain any misleading or inappropriate content. |
```

Comment:
```suggestion
| **Description** | Required for each language | The `description` field explains what your extension does, why users should install it, and any other relevant information users need to know.  <br/>* **Length:** Minimum 250 characters, maximum 5000 characters.  <br/>* **Clarity:** Must clearly describe the full functionality of your extension.  <br/>* **Completeness:** Should be thorough and include all important details.  <br/>* **Appropriateness:** Must not contain any misleading or inappropriate content. |
```

Reply:
(the change was entered w/ my other account, not this main account)